src/schemas/json/typedoc.json

```diff
@@ -0,0 +1,163 @@
+      "type": "boolean"
+    },
+    "excludePrivate": {
+      "default": false,
+      "description": "Ignores private variables and methods",
+      "type": "boolean"
+    },
+    "excludeProtected": {
+      "default": false,
+      "description": "Ignores protected variables and methods",
+      "type": "boolean"
+    },
+    "externalPattern": {
+      "description": "Define a pattern for files that should be considered being external.",
+      "type": "string"
+    },
+    "gaID": {
+      "description": "Set the Google Analytics tracking ID and activate tracking code.",
+      "type": "string"
+    },
+    "gaSite": {
+      "default": "auto",
+      "description": "Set the site name for Google Analytics.",
+      "type": "string"
+    },
+    "gitRevision": {
+      "description": "Use specified revision or branch instead of the last revision for linking to GitHub source files.",
+      "type": "string"
+    },
+    "hideGenerator": {
+      "default": false,
+      "description": "Do not print the TypeDoc link at the end of the page.",
+      "type": "boolean"
+    },
+    "ignoreCompilerErrors": {
+      "default": false,
+      "description": "Generates documentation, even if the project does not TypeScript compile.",
+      "type": "boolean"
+    },
+    "includeDeclarations": {
+      "default": false,
+      "description": "Turn on parsing of .d.ts declaration files.",
+      "type": "boolean"
+    },
+    "includes": {
+      "description": "Specifies the location to look for included documents (use [[include:FILENAME]] in comments).",
+      "type": "string"
+    },
+    "json": {
+      "description": "Specifies the location to output a JSON file containing all of the reflection data.",
+      "type": "string"
+    },
+    "listInvalidSymbolLinks": {
+      "default": false,
+      "description": "Emits a list of broken symbol [[navigation]] links after documentation generation",
+      "type": "boolean"
+    },
+    "logger": {
+      "default": "console",
+      "description": "Specify the logger that should be used.",
+      "enum": ["console", "none"]
+    },
+    "media": {
+      "description": "Specifies the location with media files that should be copied to the output directory.",
+      "type": "string"
+    },
+    "mode": {
+      "default": "modules",
+      "description": "Specifies the output mode the project is used to be compiled with.",
+      "enum": ["file", "modules"]
+    },
+    "name": {
+      "description": "Set the name of the project that will be used in the header of the template.",
+      "type": "string"
+    },
+    "out": {
+      "description": "Specifies the location the documentation should be written to.",
+      "type": "string"
+    },
+    "plugin": {
+      "description": "Specify the npm plugins that should be loaded. Omit to load all installed plugins.",
+      "oneOf": [
+        {
+          "enum": ["none"]
+        },
```

Comment:
Missed this on the first go around - when passing `none` it should still be in an array.